recipe/meta.yaml

```diff
@@ -25,7 +25,7 @@ requirements:
 
   run:
     - python >=3.7
-    - pytest >=7.0.0,<8.0.0
+    - pytest >=7.0.0,<9.0.0
```

Comment:
Actually upstream is now pinning only to >=7:

https://github.com/pytest-dev/pytest-metadata/blob/65ab3e4c3c714680e79fcf82b9795620e3f4640d/pyproject.toml#L42-L44

So I think we should follow suite here. :+1:

Reply:
@nicoddemus cheers for the lightning speed lookup, mate! Completely agreed, lemme plop the change :+1: